big-red-shuttle/EmergencyViewController.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  EmergencyViewController.swift
+//  big-red-shuttle
+//
+//  Created by Austin Astorga on 10/26/16.
+//  Copyright © 2016 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class EmergencyViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    var tableView: UITableView!
+    let emergencyArray = [("Call Cornell Univeristy Police", 6072551111), ("Call Emergency Services", 911), ("Call Blue Light Escorts", 6072557373), ("Call Gannett Health Services", 6072555155)] //add more later
```

Comment:
I'd recommend against tuples as your data structure for readability. When you are saying `.0` and `.1` all over the place, it becomes very unclear what those mean.